apps/prairielearn/src/pages/news_items/news_items.html.ts

```diff
@@ -24,8 +24,7 @@ export function NewsItems({
     urlPrefix: string;
     news_item_notification_count?: number | null;
   };
-  const timeZone =
-    resLocals.course_instance?.display_timezone ?? resLocals.course?.display_timezone ?? 'UTC';
+  const timeZone = 'UTC';
```

Comment:
This is only used in one place in this file, consider inlining it.

Reply:
Done in #9981.